### DISCUSSION TITLE
Adding feature but unable to push due to "test"'s not matching

### DISCUSSION THREAD

**Discussion by hawwokitty** [React95/React95#470] [Q&A]
Hi!!
I just made it so that if you add a minimize button to any modal, it actually works, however when I try to commit and push the changes to my github i get an error saying it doesn't match the tests. I've never really used tests before so I'm not quite sure how to get past this, please help me! 

edit:
here's how it looks btw

https://github.com/user-attachments/assets/20b596e5-2b0c-4336-8706-c965020eface

**Comment by ggdaltoso** [comment]
What an amazing feature!

And yes, we have a lot of tests to ensure code quality 👌 

You have three options:
 1. add tests for what you're doing
 2. ignore errors when pushing your changes and add tests after
    1. you can commit as usual but add a `--no-verify` tag to your `git push` command: `git push --no-verify`
 3. update the existing tests by running `yarn test:core -u`, at the root level, to update snapshots. But please, make sure to add a test case to your feature. 

If you want, [we have a room to chat](https://matrix.to/#/#React95_React95:gitter.im). I suggest the Element app; you can use it in the browser or download the Windows app. It is a safe space where we can talk and discuss details.

**Comment by ggdaltoso** [comment]
Closed by #471
